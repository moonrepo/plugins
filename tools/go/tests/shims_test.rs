use proto_pdk_test_utils::*;

mod go_tool {
    use super::*;

    #[cfg(not(windows))]
    generate_shims_test!("go-test", ["go"]);
}
