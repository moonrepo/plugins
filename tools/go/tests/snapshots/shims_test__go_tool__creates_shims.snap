---
source: tools/go/tests/shims_test.rs
expression: "std :: fs ::\nread_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "go": {},
  "gofmt": {
    "alt_bin": true,
    "parent": "go-test"
  }
}
